static/app/views/onboarding/components/useOnboardingQueryParams.tsx

```diff
@@ -0,0 +1,57 @@
+import {useCallback} from 'react';
+
+import {decodeBoolean, decodeList} from 'sentry/utils/queryString';
+import useLocationQuery from 'sentry/utils/url/useLocationQuery';
+import {useLocation} from 'sentry/utils/useLocation';
+import {useNavigate} from 'sentry/utils/useNavigate';
+
+type QueryValues = {
+  /**
+   * Used to show product selection (error monitoring, tracing, profiling and session replay) for certain platforms, e.g. javascript-react
+   */
+  product?: string[];
+  /**
+   * Used to show the loader script for when the platform is javascript
+   */
+  showLoader?: boolean;
+  /**
+   * Used to show or not the integration onboarding for certain platforms, e.g. AWS (python)
+   */
+  showManualSetup?: boolean;
```

Comment:
As there is always a fallback value they can be required, right?
```suggestion
  /**
   * Used to show product selection (error monitoring, tracing, profiling and session replay) for certain platforms, e.g. javascript-react
   */
  product: string[];
  /**
   * Used to show the loader script for when the platform is javascript
   */
  showLoader: boolean;
  /**
   * Used to show or not the integration onboarding for certain platforms, e.g. AWS (python)
   */
  showManualSetup: boolean;
```

Reply:
hm yes for the booleans but for the product there is not yet a default

Reply:
oh yes... good eyes 👀🙂